docs/resource_sets/resource_sets.rst

```diff
@@ -0,0 +1,56 @@
+**********************************
+Resource sets and partial compile
+**********************************
+To support constant scalability resource sets can be used. Without the use of those, each time a resource is added,
```

Comment:
I would not use the term "constant scalability", because the time to compile a model will not be constant. The compile time depends on the size of resource set and the amount of resources in one set depends on the user. The main difference is that the compile time depends on the size of the resource set and not the size of the entire model.